### PR TITLE
rename imageState to replicationState in the volumeReplication spec

### PR DESCRIPTION
**Pull request by sp98**
fixes: #46 

Signed-off-by: Santosh Pillai <sapillai@redhat.com>

### CODE REVIEW

**Review by Madhu-1 — APPROVED** [review]

**Comment by Madhu-1** [comment]
@sp98 Need rebase

**Review by iamniting — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ VolumeReplicationClass corresponding to the driver providing replication.
 
 `volumeReplicationClass` is the class providing replication
 
-`imageState` is the state of the image being referenced. Possible values are `primary`. `secondary` and `resync`.
+`replicationState` is the state of the image being referenced. Possible values are `primary`. `secondary` and `resync`.
```
Comment (umangachapagain) [inline README.md:47]:
```suggestion
`replicationState` is the state of the volume being referenced. Possible values are `primary`. `secondary` and `resync`.
```
If we're not mentioning "image", should we also make this change at all places?

Comment (sp98) [reply README.md]: yep. done.

**Review by Madhu-1 — APPROVED** [review]

**Review by Madhu-1 — CHANGES_REQUESTED** [review]
Need to squash the commits?

**Review by umangachapagain — APPROVED** [review]

**Review by Madhu-1 — APPROVED** [review]

**Pull request merged**